vizro-core/changelog.d/20230915_010825_maximilian_schulz_prepare_automation.md

```diff
@@ -0,0 +1,42 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
```

Comment:
Do we want to insert content here?

Reply:
I thought about it, but I think it is not really relevant to the user right?